tests/Unit/PointwiseFunctions/AnalyticSolutions/Poisson/Test_ProductOfSinusoids.cpp

```diff
@@ -74,7 +74,7 @@ void test_solution(const std::array<double, Dim>& wave_numbers,
       "ProductOfSinusoids", {"source"}, {{{0., 2. * M_PI}}},
       std::make_tuple(wave_numbers), DataVector(5));
 
-  Poisson::Solutions::ProductOfSinusoids<Dim> created_solution =
+  auto created_solution =
```

Comment:
So I understand, what's going on in this file? Why these changes?

Reply:
That is because clang-tidy had a complaint.